tests/NamingStrategy/PathNamingStrategyTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Http\Client\Plugin\Vcr\Tests\NamingStrategy;
+
+use GuzzleHttp\Psr7\Request;
+use Http\Client\Plugin\Vcr\NamingStrategy\PathNamingStrategy;
+use PHPUnit\Framework\TestCase;
+use Psr\Http\Message\RequestInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @internal
+ */
+class PathNamingStrategyTest extends TestCase
+{
+    /**
+     * @dataProvider provideRequests
+     */
+    public function testName(string $expected, RequestInterface $request, array $options = []): void
+    {
+        $strategy = new PathNamingStrategy($options);
+
+        $this->assertSame($expected, $strategy->name($request));
+    }
+
+    public function testConfigureOptions(): void
+    {
+        $resolver = new OptionsResolver();
+
+        $expected = [
+            'hostname_prefix' => false,
+            'name_prefix' => '',
+            'use_headers' => false,
+            'hash_body_methods' => ['PUT'],
+        ];
+
+        (new PathNamingStrategy())->configureOptions($resolver);
```

Comment:
we should instead have a test for each option on and off to be sure they work as expected

Reply:
I added test cases for both options.